src/applications/claims-status/components/evss/DetailsPageContent.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+export default function DetailsPageContent({ claim }) {
+  const {
+    claimType,
+    contentionList,
+    dateFiled,
+    vaRepresentative,
+  } = claim.attributes;
+
+  return (
+    <>
+      <h3 className="vads-u-visibility--screen-reader">Claim details</h3>
+      <dl className="claim-details">
+        <dt className="claim-detail-label">
+          <h4>Claim type</h4>
+        </dt>
+        <dd>{claimType || 'Not Available'}</dd>
+        <dt className="claim-detail-label">
+          <h4>What you’ve claimed</h4>
+        </dt>
+        <dd>
+          {contentionList && contentionList.length ? (
+            <ul className="claim-detail-list">
+              {contentionList.map((contention, index) => (
+                <li key={index} className="claim-detail-list-item">
+                  {contention}
+                </li>
+              ))}
+            </ul>
+          ) : (
+            'Not Available'
+          )}
+        </dd>
+        <dt className="claim-detail-label">
+          <h4>Date received</h4>
+        </dt>
+        <dd>{moment(dateFiled).format('MMM D, YYYY')}</dd>
+        <dt className="claim-detail-label">
+          <h4>Your representative for VA claims</h4>
+        </dt>
+        <dd>{vaRepresentative || 'Not Available'}</dd>
+      </dl>
```

Comment:
I think @joshkimux has been recommending to not use description lists any more.

Reply:
That's a good call, and I think @joshkimux is getting ready to do another accessibility audit of the CST if this hasn't been captured yet by one of the previous ones. My goal currently is to avoid making any changes to the CST beyond shifting the code to consume the Lighthouse Benefits Claims API instead of EVSS

Reply:
Thanks for tagging me here! Confirming what @Mottie says, we're trying to move away from description lists now. Will absolutely catch this in our second run through of CST 🥳

Reply:
<img width="624" alt="Screenshot 2023-04-27 at 1 35 12 PM" src="https://user-images.githubusercontent.com/13838621/234959667-f32777ca-d1b7-4a30-a3bc-a26fd9415f48.png">
For reference, this is the portion of the UI that @Mottie is referring to that contains description lists. The URL for this page is `/track-claims/your-claims/{claimId}/details` where `claimId` is the id of the claim you are looking to view the details for